config/helpers.ts

```diff
@@ -196,36 +196,35 @@ export function getFiles(root: string): SnippetFileInput[] {
     // Helper
     function syncRecurseThroughDirectory(dir: string) {
         fs.readdirSync(dir)
-            .forEach(file => {
-                const fullPath = path.join(dir, file);
-                const withoutExt = file.replace('.yaml', '');
-
-                /* Check for file/folder naming guidelines */
-                if (!followsNamingGuidelines(withoutExt)) {
-                    throw new Error(`Invalid name at ${chalk.bold.red(fullPath)}. Name must only contain lowercase letters, numbers, and hyphens.`);
-                }
-
-                if (fs.statSync(fullPath).isDirectory()) {
-                    syncRecurseThroughDirectory(fullPath);
-                } else {
-                    files.push(getFileMetadata(fullPath, root));
-                }
-            });
+        .filter(file => !['.DS_Store'].includes(file))
```

Comment:
Is this the Mac fix you mentioned?

Reply:
Yes, macOS creates an invisible .DS_Store file in every folder when you navigate to that folder in the file explorer UI. In this situation, my yarn build kept failing because .DS_Store doesn't match the naming guidelines. I can't get rid of the .DS_Store files, but I'm open to other solutions if we don't like this simple ignore approach.